blocks/library/html/index.js

```diff
@@ -83,6 +85,13 @@ registerBlockType( 'core/html', {
 							onChange={ ( event ) => setAttributes( { content: event.target.value } ) }
 						/>
 					}
+					{ focus &&
+					<InspectorControls key="inspector">
```

Comment:
Maybe just one more tab to fix the indentation.

Reply:
Hey @youknowriad - good catch. I just added a tab for better code clarity.